submissions/2024-06-14-OpenFF-Torsion-Multiplicity-Torsion-Drive-Coverage-Supplement-v1.0/README.md

```diff
@@ -0,0 +1,70 @@
+# OpenFF Torsion Multiplicity Torsion Drive Coverage Supplement v1.0
+
+## Description
+A torsion drive data set created to improve the coverage of both existing Sage
+2.2.0 proper torsion parameters and new parameters added through the torsion
+multiplicity project. The molecules in this data set were partly selected from
+the ChEMBL 33 database and partly generated manually to match parameters not
+covered by ChEMBL.
+
+## General Information
+
+* Date: 2024-06-14
+* Class: OpenFF TorsionDrive Dataset
+* Purpose: Improver proper torsion coverage in Sage
+* Name: OpenFF Torsion Multiplicity Torsion Drive Coverage Supplement v1.0
+* Number of unique molecules: 58
+* Number of filtered molecules: 0
+* Number of conformers: 59
+* Number of conformers per molecule (min, mean, max): 1, 3.08, 10
```

Comment:
This is minor so I'll approve to get things computing over the weekend, but I don't think `n_records` is the right property for number of total conformers -- having a mean of 3 conformers for 58 molecules should give us 150+ conformers total. Could you please update the `Number of conformers` line to the sum of conformers?